src/main/java/frc/robot/Robot.java

```diff
@@ -138,9 +143,11 @@ public void teleopPeriodic() {
 
     // Replace with limelight stuff at some point
     if (manipulator.getButton(Button.RIGHT_BUMPER)) {
-      ballChucker9000.moveRotator(ballChucker9000.getRotatorEncoder() >= 270 ? 0 : 0.1);
+      // ballChucker9000.moveRotator(ballChucker9000.getRotatorEncoder() >= 270 ? 0 : 0.5);
+      ballChucker9000.moveRotator(0.5);
     } else if (manipulator.getButton(Button.LEFT_BUMPER)) {
-      ballChucker9000.moveRotator(ballChucker9000.getRotatorEncoder() <= 0 ? 0 : -0.1);
+      // ballChucker9000.moveRotator(ballChucker9000.getRotatorEncoder() <= 0 ? 0 : -0.5);
+      ballChucker9000.moveRotator(-0.5);
```

Comment:
I do not love commented code. It can stay iff there is a good reason to keep it as a comment and a comment for why we are keeping the code is added.